src/components/common/post/section/child/PostSectionChild.jsx

```diff
@@ -0,0 +1,30 @@
+import s from "./PostSectionChild.module.css";
+import {ORIGIN_URL} from "../../../../../constants/pathConstants.js";
+
+/* eslint-disable react/prop-types */
+function PostSectionChild(props) {
+    const sectionChild = props.sectionChild;
+
+    let element;
+    if (sectionChild.type === 'image') {
```

Comment:
@boandriy , can we improve this code somehow?

Reply:
Need a call hare.